framework/decode/dx12_dump_resources.cpp

```diff
@@ -2563,6 +2592,7 @@ void DefaultDx12DumpResourcesDelegate::WriteResource(nlohmann::ordered_json&   j
     util::FieldToJson(jdata["heap_index"], resource_data->descriptor_heap_index, json_options_);
     util::FieldToJson(jdata["res_id"], resource_data->source_resource_id, json_options_);
     util::FieldToJson(jdata["dimension"], util::ToString(resource_data->desc.Dimension), json_options_);
+    util::FieldToJson(jdata["modifiable_subresources_mask"], subResourceModifiedMask, json_options_);
```

Comment:
`subResourceModifiedMask` is a integer to indicate the all subresources that are modifiable or not. I feel it might be hard to read.  I think we could just write a boolean into .json inside `for (const auto sub_index : resource_data->subresource_indices)`.